editor/src/document/document_file.rs

```diff
@@ -1063,6 +1068,18 @@ impl MessageHandler<DocumentMessage, &InputPreprocessor> for DocumentMessageHand
 			SetSnapping(new_status) => {
 				self.snapping_enabled = new_status;
 			}
+			ZoomCanvasToFitAll => {
+				if let Some(bounds) = self.document_bounds() {
+					responses.push_back(
+						MovementMessage::FitViewportToBounds {
+							bounds,
+							padding_scale_factor: Some(1.05),
```

Comment:
Let's move this into `consts.rs` as `VIEWPORT_ZOOM_TO_FIT_PADDING_SCALE_FACTOR`.